src/Tasks/ResolveManifestFiles.cs

```diff
@@ -775,7 +782,15 @@ public void Add(ITaskItem item)
                 string fusionName = item.GetMetadata(ItemMetadataNames.fusionName);
                 if (String.IsNullOrEmpty(fusionName))
                 {
-                    fusionName = Path.GetFileNameWithoutExtension(item.ItemSpec);
+                    string destSubDir = item.GetMetadata(ItemMetadataNames.destinationSubDirectory);
+                    if (!String.IsNullOrEmpty(destSubDir))
+                    {
+                        fusionName = Path.Combine(destSubDir, Path.GetFileNameWithoutExtension(item.ItemSpec));
```

Comment:
Is `fusionName` a bad variable name here? Because I wouldn't expect a path in a Fusion name.

Reply:
Right, it's a confusing name but since the existing code was already using that variable name when falling back to use filename in absence of FusionName metadata, I didn't change it. Can make the name change if you feel strongly about it.

Reply:
Not required for this servicing fix, but I'd like to see it soonish for the sake of future maintainability.